recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set version = "0.6.18" %}
 # see github.com/conda-forge/conda-forge.github.io/issues/1059 for naming discussion
-{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+{% set torch_proc_type = "cuda" ~ (cuda_compiler_version | replace('.', '')) if cuda_compiler_version != "None" else "cpu" %}
```

Comment:
why are you changing this?

Reply:
So this wasn't technically needed for this fix, but essentially what I'm trying to achieve here is to ensure the pytorch cuda version is also consistent.

E.g. for cuda 12.6, previously:
```
- pytorch =*=cuda*
```
Now:
```
- pytorch =*=cuda126*
```

Should I revert this and leave it up to the env solver to get it consistent?

Reply:
Pytorch only has one 12.x CUDA version, and we don't depend on the minor version here. If we wanted to use CUDA 12.8 here, that would be fine too. So let's not do that.